templates/common/infra/bicep/core/host/container-app-upsert.bicep

```diff
@@ -0,0 +1,50 @@
+param name string
+param location string = resourceGroup().location
+param tags object = {}
+
+param containerAppsEnvironmentName string
+param containerName string = 'main'
+param containerRegistryName string
+param secrets array = []
+param env array = []
+param external bool = true
+param targetPort int = 80
+param exists bool
```

Comment:
@jongio - I see 2 paths forward here.

1. Expect a param for each service image name and use that value explicitly or a default if not set.
2. Use `existing` resource with some identifier for whether it is a safe operation to perform

Option 1 is easier but also allows container images to get set to any image at provision time and not just during `azd deploy`.  That may be good for some use cases but opens could also lead to bugs

Option 2 is more work, but if leveraging our bicep modules this would hide any of that complexity.